doc/rvv-intrinsic-spec.adoc

```diff
@@ -0,0 +1,517 @@
+
+
+[[control-to-policy]]
+=== Control of `vta`, `vma`
+
+The behavior of destination tail elements and destination inactive masked-off elements is controled by the `vta` and `vma` bits ^6^.
+
+Given the general assumption that target audience of the RVV intrinsics are high performance cores, and an "undisturbed" policy will generally slow down an out-of-order core, the intrinsics have a default policy scheme of tail-agnostic and mask-agnostic (that is, `vta=1` and `vma=1`).
+
+Please see <<policy-naming-scheme>> for the exact suffix that specifies the policy behavior of a vector operation.
+
+=== Control of `vxrm`
+
+For the fixed-point intrinsics, representing the instructions under v-spec section 12, the `vxrm` operand of the intrinsics indicates the rounding mode (`vxrm`) ^8^ control.
+
+The `vxrm` operand is required to be an immediate. The compiler should implement the following enum that maps to the defined rounding mode values under v-spec Table 4 ^8^.
+
+```c
+enum __RISCV_VXRM {
+  __RISCV_VXRM_RNU = 0,
+  __RISCV_VXRM_RNE = 1,
+  __RISCV_VXRM_RDN = 2,
+  __RISCV_VXRM_ROD = 3,
+};
+```
+
+NOTE: Computations of `vsadd`, `vsaddu`, `vssub`, and `vssubu` do not need the rounding mode, therefore the intrinsics of these instructions do not have the `vxrm` parameter.
+
+NOTE: The RISC-V psABI ^9^ states that `vxrm` is not preserved across calls. Optimization for reducing the number of redundant writes to `vxrm` is a compiler and system specific issue.
+
+[[chapter-control-to-frm]]
+=== Control of `frm`
+
+For the floating-point intrinsics, representing the instructions under v-spec section 13, the intrinsics have two classes, called implicit-`frm` and explicit-`frm`.
+
+==== Implicit-`frm` intrinsics
+
+The implicit-`frm` intrinsics behave like any C-language floating-point expressions, using the default rounding mode when `FENV_ACCESS` is off, and using the `fenv` dynamic rounding mode when `FENV_ACCESS` is on.
+
+NOTE: Both GNU and LLVM compilers generate scalar floating-point instructions using dynamic rounding mode, relying on the kernel initialization to set `frm` to `RNE` (specified as "roundTiesToEven" in IEEE-754 (a.k.a. IEC 60559)).
+
+NOTE: The implicit-`frm` intrinsics are intended to be used regardless of `FENV_ACCESS`. They are provided when `FENV_ACCESS` is on for the (few) programmers who are already using fenv. And they are provided when `FENV_ACCESS` is off for the (vast majority of) programmers who prefer the default rounding mode.
+
+[[chapter-explicit-frm]]
+==== Explicit-`frm` intrinsics
+
+The explicit-`frm` intrinsics contain the `frm` operand which indicates the rounding mode (`frm`) ^10^ control. The floating-point intrinsics with the `frm` operand is followed by an `_rm` suffix in the function name.
+
+The `frm` operand is required to be an immediate. The compiler should implement the following enum that maps to the defined rounding mode values under RISC-V ISA Manual Table 8.1 ^8^.
```

Comment:
immediate -> "compile time constant" or maybe "integer constant expression" if you want to get real technical?